server/app/services/geo/esri/EsriClient.java

```diff
@@ -309,8 +177,8 @@ public CompletionStage<ImmutableList<ServiceAreaInclusion>> getServiceAreaInclus
             (maybeJson) -> {
               if (maybeJson.isEmpty()) {
                 logger.error(
-                    "EsriClient.fetchServiceAreaFeatures JSON response is empty. Called by"
-                        + " EsriClient.isAddressLocationInServiceArea."
+                    "RealEsriClient.fetchServiceAreaFeatures JSON response is empty. Called by"
+                        + " RealEsriClient.isAddressLocationInServiceArea."
```

Comment:
Same as above comments on the logger error message about `RealEsriClient`

Reply:
updated